bblfsh/compat.py

```diff
@@ -0,0 +1,228 @@
+import os
+import sys
+from typing import Union, List, Any, Optional
+
+import grpc
+
+import bblfsh.client as bcli
+from bblfsh import role_id, role_name
+from bblfsh.node import Node
+from bblfsh.node_iterator import NodeIterator
+from bblfsh.result_context import ResultContext
+from bblfsh.aliases import (
+    ParseRequest, ParseResponse, DriverStub, ProtocolServiceStub,
+    VersionRequest, SupportedLanguagesRequest, ModeType,
+    Mode, VersionResponse, DESCRIPTOR
+)
+from bblfsh.pyuast import uast, iterator as native_iterator
+from bblfsh.tree_order import TreeOrder
+
+if "BBLFSH_COMPAT_SHUTUP" not in os.environ:
+    print("Warning: using deprecated bblfsh v1 compatibility layer.",
+          file=sys.stderr)
+
+
+class WrongTypeException(Exception):
```

Comment:
Please add at least brief doc-strings to this and the other classes below. For exceptions I think the interesting datum is where the exception comes from (e.g., "this is thrown from queries when the API returns a type that doesn't match the request" or whatever is actually true).

Reply:
Sure.

Reply:
Done!